charts/aws-fluent-bit/templates/clusterrole.yaml

```diff
@@ -18,3 +19,24 @@ rules:
       - get
       - list
       - watch
+  {{- if and .Values.podSecurityPolicy.create (semverCompare "<=1.25-0" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
@patrick-stephens 
I think this is not rendering as we expect to. 
I have set the  **.Values.podSecurityPolicy.create** to _true_ and I'm on the k8s version 1.25 but i still get to see the podSecurityPolicy created

Reply:
Not sure I follow: do you see it on 1.25 as that is expected with `<=`?

Reply:
I think PodSecurityPolicies are not allowed in 1.25 version
but i see that the podSecurityPolicy is created.